CrowdFundingApplication/Startup.cs

```diff
@@ -48,7 +48,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllerRoute(
                     name: "default",
-                    pattern: "{controller=Home}/{action=Index}/{id?}");
+                    pattern: "{controller=Home}/{action=MainPage}/{id?}");
```

Comment:
Why this change?

Reply:
When a user for the first time on this web-app or unauthorized we allow use only welcome page where the user can register or authorize. When the user is already registered in web-app this is unnecessary to login again and we allow the user start with MainPage